### DISCUSSION TITLE
When will RSuite support React 18？

### DISCUSSION THREAD

**Discussion by vingeraycn** [rsuite/rsuite#2429] [Q&A] [answered]
React 18 has released, when will RSuite support React 18？

**Comment by SevenOutman** [comment]
Hopefully there won't be many compatibility issues. We'll update the peerDependency after we've tested rsuite with react 18.

**Reply by vingeraycn** [reply]
ToasterContainer has compatibility issue as I know.

**Comment by simonguo** [comment, marked as answer]
Adapt to React 18. https://github.com/rsuite/rsuite/pull/2595


Install the beta version that supports react 18.
```
npm i rsuite@next
```

**Reply by simonguo** [reply]
Supported in version [5.21.0](https://github.com/rsuite/rsuite/releases/tag/v5.21.0)
